src/components/Dashboard/Alerts/MissingInformation.vue

```diff
@@ -33,17 +37,26 @@
 <script lang="ts">
 import Vue from 'vue'
 import { Component } from 'vue-property-decorator'
+import { Getter } from 'vuex-class'
 import { ContactInfo } from '@/components/common'
+import { getFeatureFlag } from '@/utils'
 
 @Component({
   components: { ContactInfo }
 })
 export default class MissingInformation extends Vue {
-  protected panel = 1
+  @Getter isFirm!: boolean
 
-  protected togglePanel (): void {
+  private panel = 1
+
+  private togglePanel (): void {
```

Comment:
In Vue 2.7, with latest Volar extension, properties and local methods are now accessible in the template when they are private.